### PR TITLE
Added Plastic into the Autolathe 

### PR DESCRIPTION
**Pull request by SethLafuente**

### CODE REVIEW

**Comment by SweptWasTaken** [comment]
Why did you delete all the PR template? Did you read it?

File: Content.Server/Cargo/CargoOrderDatabase.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public CargoOrderDatabase(int id)
         {
             Id = id;
             CurrentOrderSize = 0;
-            MaxOrderSize = 20;
```
Comment (SweptWasTaken) [inline Content.Server/Cargo/CargoOrderDatabase.cs:18]: Why did you change this?

File: Content.Server/GameObjects/Components/Nutrition/FoodComponent.cs [hunk 2]
```diff
@@ -167,7 +167,7 @@ public virtual bool TryUseFood(IEntity? user, IEntity? target, UtensilComponent?
                 SoundSystem.Play(Filter.Pvs(trueTarget), UseSound, trueTarget, AudioParams.Default.WithVolume(-1f));
             }
 
-            trueTarget.PopupMessage(user, Loc.GetString("Nom"));
```
Comment (SweptWasTaken) [inline Content.Server/GameObjects/Components/Nutrition/FoodComponent.cs:170]: Why did you change this?

**Review by SweptWasTaken — APPROVED** [review]

File: Content.Server/GameObjects/Components/Nutrition/FoodComponent.cs (hunk 2 above)
Comment (RemieRichards) [reply Content.Server/GameObjects/Components/Nutrition/FoodComponent.cs:170]: For reference, "Nom" is not a typo of "No", it's a common onomatopoeia for eating.

File: Content.Server/Cargo/CargoOrderDatabase.cs (hunk 1 above)
Comment (SethLafuente) [reply Content.Server/Cargo/CargoOrderDatabase.cs:18]: O yeah first time accessing this stuff and wasen't sure which files I had to change to make a change into the game so I made a small change to something to test if what I changed actually did something I was trying to figure out what anything was for so long that I forgot about that but now I know that this post I made contains the changes I made so thats nice

File: Content.Server/GameObjects/Components/Nutrition/FoodComponent.cs (hunk 2 above)
Comment (SethLafuente) [reply Content.Server/GameObjects/Components/Nutrition/FoodComponent.cs:170]: same thing as the other one testing if the changes I did changed something in the game

File: Content.Server/Cargo/CargoOrderDatabase.cs (hunk 1 above)
Comment (SweptWasTaken) [reply Content.Server/Cargo/CargoOrderDatabase.cs:18]: Ok go back and change it so it's not in the PR.

Comment (SethLafuente) [reply Content.Server/Cargo/CargoOrderDatabase.cs:18]: k

**Review by PaulRitter — CHANGES_REQUESTED** [review]
change the issues swept noted to get merged. just putting a request changes on this so i dont get confused since swept approved it

File: Resources/Prototypes/Entities/Constructible/Power/lathe.yml [hunk 3]
```diff
@@ -135,6 +135,7 @@
       - LightBulb
       - SheetSteel
       - GlassStack
+      - SheetPlastic
       - CableStack
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Constructible/Power/lathe.yml:139]: At least try and order it :\

**Comment by Visne** [comment]
good first pr

File: Resources/Prototypes/Entities/Constructible/Power/lathe.yml (hunk 3 above)
Comment (SethLafuente) [reply Resources/Prototypes/Entities/Constructible/Power/lathe.yml:139]: Yeah I will change it in a few hours also didn’t think the order mattered when I looked at them

**Comment by metalgearsloth** [comment]
Superseded by https://github.com/space-wizards/space-station-14/pull/4039 now
